examples/http-retry/http_retry.client.out

```diff
@@ -1,5 +1,4 @@
-# Once you send a request to the retryDemoService requests will be retried if the there is
-# any error occurred during the network communication.
+# If the request that was sent to the `retryDemoService` failes due to an error, the client tries sending the request again. 
```

Comment:
@shavidissa word failes should rename to fails right??

Reply:
Correct. Thank you for catching that. Sent another commit.